src/components/molecules/reviewCard/StarRating.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from 'next/image';
+import assets from '@/variables/images';
+
+interface StarRatingProps {
+  starNumber?: number;
+}
+
+const StarRating = ({ starNumber = 0 }: StarRatingProps) => {
+  const length = Math.max(1, Math.min(5, Math.floor(starNumber)));
+
+  return (
+    <div className='flex'>
+      {Array.from({ length }, (_, i) => (
+        <Image
+          key={`full-${i}`}
```

Comment:
```suggestion
          key={i}
```
`starNumber`가 소수점이어도 `Math.floor(starNumber)`에 의해 무조건 내림이 적용되어 정수 부분만 반환되는 것 같고,
또한 `emptyStar`를 추가하는 로직이 따로 보이지 않는데 혹시 이후에 따로 사용하실 계획이 있나요?!

만약 없다면(!!!) `key`가 리스트 내에서만 유니크하면 충분하기 때문에 `full-` 접두어를 붙이지 않고
`key={i}`만 사용해도 충분하지 않을까요? 

이렇게 하면 템플릿 리터럴을 사용하지 않아도 돼서 불필요한 문자열 연산을 줄이는 데도 도움이 될 것 같아요!

Reply:
아 맞네요 수정했습니다~!